deployments/helm-chart/values.yaml

```diff
@@ -474,8 +461,26 @@ prometheus:
     ## Requires prometheus.create=true
     create: false
 
+    ## Name of the endpoint exposed by the service.
+    ## Used by the ServiceMonitor's `endpoints` field to discover service endpoints.
+    endpoint: "prometheus"
+
     labels: {}
 
+  serviceMonitor:
+    ## Creates a serviceMonitor to expose statistics on the kubernetes pods.
+    create: false
+
+    ## Kubernetes object labels to attach to the serviceMonitor object.
+    labels: {}
+
+    ## A set of labels to allow the selection of endpoints for the ServiceMonitor.
+    selectorMatchLabels: {}
```

Comment:
I don't know if we even need to have these customizable. And the `labels` in the `service`.
I think we can remove the extra fields and set the labels in the yaml files directly. I can't see why somebody would want to customize these lables...

Reply:
I feel it is still useful to keep this customizable.
I think it would be good if we have a default label selection there so if users just do a helm install without setting any arguments it will still work out of the box.